src/components/Conversion/ConversionNOB.vue

```diff
@@ -143,12 +144,14 @@ export default class NatureOfBusiness extends Mixins(CommonMixin) {
         })
       }
       this.onEditMode = false
+      this.hasEdited = true
```

Comment:
Should this be True only if NAICS has changed?

Did you test that this works after resuming a draft correction with a NAICS change?

Reply:
Yes. This variable is created specifically for this purpose.